docs/requirements.txt

```diff
@@ -0,0 +1,5 @@
+Sphinx>4.3
+sphinx-rtd-theme>=1
+ZConfig
+Zope
+tempstorage
```

Comment:
This should be `./doc-requirements.txt` or the contents of the `docs` extra in setup.py

Reply:
done